consensus/polybft/command/polybftsecrets/params.go

```diff
@@ -0,0 +1,207 @@
+package polybftsecrets
```

Comment:
This entire package should not be in the `consensus` package, but in the `command` package - please move it out

Reply:
It was an intentional decision to not integrate it with secrets init yet. We expect to follow better guidelines when integrating with edge code that it is already in production and this PR was meant to be a WIP.